WIP PR for SRVLOGIC-209:  Add the "Openshift Serverless Logic overview" section. (#70393)
openshift/openshift-docs: about/overview-openshift-serverless-logic.adoc

:_content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="overview-openshift-serverless-logic"]
= Openshift Serverless Logic overview
:context: overview-openshift-serverless-logic

toc::[]

> OpenShift Serverless Logic enables developers or architects to define declarative workflow models that orchestrate event-driven, serverless applications.

Reviewer: ```suggestion
OpenShift Serverless Logic enables developers or architects to create cloud-native applications using declarative workflow models that orchestrate event-driven, serverless applications.
```

nitpick, kind of missing the notion of cloudnativeapps, with my suggestion it feels too long thought so I am not sure.

Reviewer 2: I think that the original text here is good, because it introduces the main concept of "workflows" to do, bla, bla.

> OpenShift Serverless Logic is also designed to write workflows in formats (JSON or YAML) that might be better suited for developing and deploying serverless applications in cloud or container environments.

Reviewer: ```suggestion
OpenShift Serverless Logic allows users to define workflows in formats (JSON or YAML) that are well suited for developing and deploying serverless applications in cloud or container environments. 
```
wdyt?

> For OpenShift Serverless Logic implementation, you can use the Kubernetes operator to deploy the workflow models using Knative. Knative Eventing plays an important role and provides the essential infrastructure for event-driven architectures.

Reviewer: ```suggestion
OpenShift Serverless Logic provides the Kubernetes operator to deploy & manage the workflow models using Knative. Knative Eventing plays an important role and provides the essential infrastructure for event-driven architectures.
```
Small suggestion, let me know.
We should also mentions serving somehow imho.
Lastly, from user perspective it did not tell me much, if we are talking overview I think it could be expanded slightly.

Reviewer 2: Maybe we can say something like this:

OpenShift Serverless Logic provides the Kubernetes operator to deploy & manage the workflow models using Knative Serving or Kubernetes deployments.  And, Knative Eventing provides the essential infrastructure for event-driven architectures.
